extensions/scarb-cairo-execute/src/main.rs

```diff
@@ -0,0 +1,185 @@
+
+    match main_inner(args, ui.clone()) {
+        Ok(()) => ExitCode::SUCCESS,
+        Err(error) => {
+            ui.error(format!("{error:#}"));
+            ExitCode::FAILURE
+        }
+    }
+}
+
+fn main_inner(args: Args, ui: Ui) -> Result<(), anyhow::Error> {
+    let metadata = MetadataCommand::new().inherit_stderr().exec()?;
+    let package = args
+        .packages_filter
+        .match_one(&metadata)
+        .map_err(|e| anyhow::anyhow!("Failed to match package in workspace: {e:?}"))?;
+
+    let filename = format!("{}.executable.json", package.name);
+    let path = Utf8PathBuf::from(env::var("SCARB_TARGET_DIR")?).join(env::var("SCARB_PROFILE")?);
+
+    ui.print(Status::new("Running", &package.name));
+    let executable = load_prebuilt_executable(&path, filename)?;
+
+    let data = executable
+        .program
+        .bytecode
+        .iter()
+        .map(Felt252::from)
+        .map(MaybeRelocatable::from)
+        .collect();
+
+    let (hints, string_to_hint) = build_hints_dict(&executable.program.hints);
+
+    let program = {
+        let entrypoint = executable
+            .entrypoints
+            .iter()
+            .find(|e| matches!(e.kind, EntryPointKind::Bootloader))
+            .ok_or_else(|| anyhow::anyhow!("Bootloader entrypoint not found"))?;
+        Program::new(
+            entrypoint.builtins.clone(),
+            data,
+            Some(entrypoint.offset),
+            hints,
+            Default::default(),
+            Default::default(),
+            vec![],
+            None,
+        )
+    };
+
+    let mut hint_processor = CairoHintProcessor {
+        runner: None,
+        user_args: vec![vec![Arg::Array(
+            args.run.args.iter().map(|v| Arg::Value(v.into())).collect(),
+        )]],
+        string_to_hint,
+        starknet_state: Default::default(),
+        run_resources: Default::default(),
+        syscalls_used_resources: Default::default(),
+        no_temporary_segments: false,
+    };
+
+    let cairo_run_config = CairoRunConfig {
+        trace_enabled: true,
+        relocate_mem: false,
+        layout: LayoutName::all_cairo,
+        proof_mode: false,
+        secure_run: Some(true),
+        allow_missing_builtins: Some(true),
+        ..Default::default()
+    };
+
+    let mut runner = cairo_run_program(&program?, &cairo_run_config, &mut hint_processor)
+        .map_err(|e| anyhow::anyhow!("Cairo program run failed: {e:?}"))?;
+
+    if args.run.print_outputs {
+        let mut output_buffer = "Program Output:\n".to_string();
+        runner.vm.write_output(&mut output_buffer)?;
+        print!("{output_buffer}");
+    }
+
+    let file_path = create_incremental_file_in_dir("target/cairo-execute", Some(".zip"))?;
+    runner.get_cairo_pie()?.write_zip_file(&file_path)?;
+
+    Ok(())
+}
+
+fn load_prebuilt_executable(
+    path: &Utf8PathBuf,
+    filename: String,
+) -> Result<Executable, anyhow::Error> {
```

Comment:
```suggestion
) -> anyhow::Result<Executable> {
```

Please import `anyhow::Result` too, instead of adding prefix.